tests/legacy-cli/e2e_runner.ts

```diff
@@ -138,6 +146,8 @@ if (testsToRun.length == allTests.length) {
   console.log(`Running ${testsToRun.length} tests (${allTests.length} total)`);
 }
 
+console.log('Tests: \n  ', testsToRun.join('\n  '));
```

Comment:
I see what you are trying to do here, but actually this causes incorrect indent for the first item.

<img width="462" alt="Screenshot 2022-07-28 at 09 25 28" src="https://user-images.githubusercontent.com/17563226/181446502-d1b6ecb8-8a5b-41ab-b587-e71d934eb977.png">
Two ways can can address this

```ts
console.log('Tests:', testsToRun.join('\n  '));  
```
or
```ts
console.log(['Tests:', ...testsToRun].join('\n '));
```

Reply:
Good catch 👍